src/Tools/dotnet-dev-certs/src/Program.cs

```diff
@@ -363,9 +363,10 @@ private static int EnsureHttpsCertificate(CommandOption exportPath, CommandOptio
             {
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
                 {
+                    var certPath = exportPath.HasValue() ? exportPath.Value() : "<<certificate>>";
                     reporter.Warn("Trusting the HTTPS development certificate was requested. If the certificate is not " +
                         "already trusted we will run the following command:" + Environment.NewLine +
-                        "'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'" +
+                        $"'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain {certPath}'" +
```

Comment:
How does this fix the issue?

I would suggest that the message instead becomes something like: "To undo these changes ...." and suggest either running `dotnet dev-certs https --clean` to remove the cert (and the trust) or to display the right command on mac os to remove the certificate from the list of trusted system certificates.

Reply:
So just add a sentence on the end that says "To undo these changes, run dotnet dev-certs https --clean" and leave it as `<<cert>>` always?

Reply:
Okay updated it to add: `To undo these changes: 'sudo security remove-trusted-cert -d <<certificate>>'`